index.html

```diff
@@ -4297,19 +4304,60 @@ <h2>JSON Processing</h2>
 implementing the rule above. This can ensure proper term identification,
 typing, and order, when a JSON document is processed as JSON-LD.
           </p>
+          <p>
+However, the intention of the issuer is not preserved, unless the <code>@context</code>
+is applied, and the resulting <a href="#verifiable-credential-graphs">Verifiable Credential Graphs</a>,
+and their exact RDF types, and node and edge structure is understood by the verifier.
+          </p>
 
           <p>
-The rule above guarantees semantic interoperability between JSON and JSON-LD for
-literal JSON keys mapped to URIs by the `@context` mechanism. While JSON-LD
-processors will use the specific mechanism provided and can verify that all
-terms are correctly specified, JSON-based processors implicitly accept the same
-semantics without performing any JSON-LD transformations, but instead by
+These rules alone do not guarantee semantic interoperability between JSON and JSON-LD for
+literal JSON keys mapped to URIs by the `@context` mechanism.
+          </p>
+While some RDF processors will use the language features of JSON-LD and can verify that all
```

Comment:
Ran out of time down here -- will have to do more review later.

Reply:
Please re-review, I've applied some suggestions from Manu, and the discussion on them is relevant to your suggestions which I have not applied.

As I see it the main challenge remaining is to address the "hard coded" vs "errata published" scenario for contexts, in the case where the issue does not intend to equivocate but is forced to by their commitment to a context hosted by a third party (including W3C as a 3rd party).